cloudify-services/values.yaml

```diff
@@ -193,12 +167,17 @@ rest_service:
       successThreshold: 1
       failureThreshold: 3
   configPath: /tmp/config.yaml
+  s3:
+    credentials_secret_name: seaweedfs-s3-secret
+    session_token_secret_name: ""
   config:
     manager:
       hostname: cloudify-manager
       private_ip: localhost
       public_ip: localhost
       file_server_type: s3
+      s3_resources_bucket: resources
+      s3_server_url: http://seaweedfs-s3.default.svc.cluster.local:8333
```

Comment:
this will need to stop hardcoding `default` but I currently don't know how to do it